src/bundle/Resources/views/themes/admin/content/create/create.html.twig

```diff
@@ -70,14 +75,6 @@
     {{ knp_menu_render(content_create_sidebar_right, {'template': '@ezdesign/ui/menu/sidebar_right.html.twig'}) }}
 {% endblock %}
 
-{% block close_button %}
```

Comment:
You cannot remove this block. A lot of templates extend this one and override this block `close_button` for example page builder, content on the fly and so